packages/ariakit/src/combobox/__examples__/combobox-typing-only-popover/test.tsx

```diff
@@ -0,0 +1,48 @@
+import { click, getByRole, press, render } from "ariakit-test";
+import { axe } from "jest-axe";
+import Example from ".";
+
+const getCombobox = () => getByRole("combobox");
+const getPopover = () => getByRole("listbox", { hidden: true });
+const getOption = (name: string) => getByRole("option", { name });
+
+test("ally", async () => {
+  const { container } = render(<Example />);
+  expect(await axe(container)).toHaveNoViolations();
+});
+
+describe("combobox is pristine", () => {
+  test("does not show on click", async () => {
+    render(<Example />);
+    expect(getPopover()).not.toBeVisible();
+    await click(getCombobox());
+    expect(getPopover()).not.toBeVisible();
+  });
+
+  test("does not show on arrow down key", async () => {
+    render(<Example />);
+    await press.Tab();
+    expect(getPopover()).not.toBeVisible();
+    await press.ArrowDown();
+    expect(getPopover()).not.toBeVisible();
+  });
+});
+
+describe("combobox is dirty", () => {
+  test("show on click", async () => {
+    render(<Example defaultValue="a" />);
```

Comment:
I've added the defaultValue prop to avoid the ceremony of typing the combobox, loose focus and click.